examples/wip/viewport-animation-first-person-map/app.js

```diff
@@ -0,0 +1,346 @@
+/* global window, document, fetch */
+import React, {Component} from 'react';
```

Comment:
Nit: Do we need two examples? Can't we just have one?

Reply:
I will merge them into one in future PR.